packs/chatops/actions/chatops_itests.meta.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: "chatops_itests"
+description: "Chatops integration tests."
+runner_type: "action-chain"
+enabled: true
+entry_point: "workflows/chatops_itests.yaml"
+parameters:
+    token:
+      type: "string"
+      description: "st2 auth token"
+      default: ""
+    repos_base:
+      type: "string"
+      description: "Base for all repos to be cloned to"
+      default: "/home/stanley"
+    st2_repo:
+      type: "string"
+      description: "Url of the st2 repo to clone"
+      default: "https://github.com/StackStorm/st2.git"
```

Comment:
Always nicers to use the ssh urls.

We can always setup the server with the right keys for `stanley`.